sync/sync-impl/src/main/java/com/duckduckgo/sync/impl/SyncRepository.kt

```diff
@@ -211,9 +237,13 @@ data class AccountInfo(
     val secretKey: String = "",
 )
 
+data class LinkCode(
+    val recovery: RecoveryCode,
+)
+
 data class RecoveryCode(
-    val primaryKey: String,
-    val userID: String,
+    val primary_key: String,
```

Comment:
I really don't like to leak the API terrible field names into our code.

```suggestion
  @Json(name = "primary_key) val primaryKey: String
```